src/beast/beast/asio/streambuf.h

```diff
@@ -49,11 +51,19 @@ class basic_streambuf
     class element;
 
     using alloc_traits = std::allocator_traits<Allocator>;
```

Comment:
Will fix. This is addressed correctly in a later PR, we rebind to `std::uint8_t`